firmware/examples/SdCard/source/main.cpp

```diff
@@ -0,0 +1,60 @@
+#include <cstdint>
+
+#include "L3_HAL/sddriver.hpp"
+#include "utility/log.hpp"
+#include "utility/time.hpp"  // halt()
+#include "utility/debug.hpp"  // debug::Hexdump()
+
+const uint8_t kLongText[512] = R"(Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit es se cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat no n proident, sunt in culpa qui officia deserunt mollit anim id est laborum.)";
```

Comment:
Hmmm, it seems that clang-format did not modify this data. Probably because it understands that this is a raw string and ignored it. Add the newlines to keep this within the 80 columns limit.

Reply:
Updating my staging branch to implement this.